examples/rust-data-loader/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "example-foxglove-ndjson-data-loader",
+  "displayName": "Example Foxglove ndjson data loader",
+  "description": "",
+  "publisher": "Foxglove",
+  "version": "1.1.0",
+  "license": "MIT",
+  "main": "./dist/extension.js",
+  "scripts": {
+    "build": "foxglove-extension build",
+    "build:wasm": "cd rust && cargo build --release --target wasm32-unknown-unknown",
+    "foxglove:prepublish": "npm run build:wasm && foxglove-extension build --mode production",
```

Comment:
Should `build` do the Rust build too? I suppose `package` handles everything so I guess it doesn't matter. I just intuitively tried to run `npm run build` after copying the example.